### PR TITLE
Add a Grafana dashboard

### PR DESCRIPTION
**Pull request by thouveng**

### CODE REVIEW

**Comment by SwannCroiset** [comment]
A grafana dashboard for grafana right :) ?

**Comment by thouveng** [comment]
:rofl: yes exactly

File: grafana/files/grafana_dashboards/grafana_influxdb.json [hunk 1]
```diff
@@ -0,0 +1,687 @@
...
+        {
+          "cacheTimeout": null,
+          "colorBackground": false,
+          "colorValue": false,
+          "colors": [
+            "rgba(245, 54, 54, 0.9)",
+            "rgba(237, 129, 40, 0.89)",
+            "rgba(50, 172, 45, 0.97)"
+          ],
+          "datasource": "lma",
```
Comment (simonpasquier) [inline grafana/files/grafana_dashboards/grafana_influxdb.json]: should be null to use the default datasource.

File: grafana/files/grafana_dashboards/grafana_influxdb.json [hunk 2]
```diff
@@ -0,0 +1,687 @@
...
+              "text": "N/A",
+              "value": "null"
+            }
+          ],
+          "valueName": "current"
+        },
+        {
+          "aliasColors": {},
+          "bars": false,
+          "datasource": "lma",
```
Comment (simonpasquier) [inline grafana/files/grafana_dashboards/grafana_influxdb.json]: ditto

File: grafana/files/grafana_dashboards/grafana_influxdb.json [hunk 3]
```diff
@@ -0,0 +1,687 @@
...
+              "max": null,
+              "min": null,
+              "show": true
+            }
+          ]
+        },
+        {
+          "aliasColors": {},
+          "bars": false,
+          "datasource": "lma",
```
Comment (simonpasquier) [inline grafana/files/grafana_dashboards/grafana_influxdb.json]: ditto

File: grafana/files/grafana_dashboards/grafana_influxdb.json [hunk 4]
```diff
@@ -0,0 +1,687 @@
...
```
Comment (simonpasquier) [inline grafana/files/grafana_dashboards/grafana_influxdb.json]: ditto

File: grafana/files/grafana_dashboards/grafana_influxdb.json [hunk 5]
```diff
@@ -0,0 +1,687 @@
...
+  "sharedCrosshair": true,
+  "style": "dark",
+  "tags": [],
+  "templating": {
+    "list": [
+      {
+        "allFormat": "glob",
+        "allValue": null,
+        "current": {},
+        "datasource": "lma",
```
Comment (simonpasquier) [inline grafana/files/grafana_dashboards/grafana_influxdb.json]: ditto

File: grafana/files/grafana_dashboards/grafana_influxdb.json [hunk 6]
```diff
@@ -0,0 +1,687 @@
...
+        "sort": 0,
+        "tagValuesQuery": null,
+        "tagsQuery": null,
+        "type": "query"
+      },
+      {
+        "allFormat": "glob",
+        "allValue": null,
+        "current": {},
+        "datasource": "lma",
```
Comment (simonpasquier) [inline grafana/files/grafana_dashboards/grafana_influxdb.json]: ditto

**Comment by thouveng** [comment]
done

**Pull request merged**